src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -270,12 +275,49 @@ public void ErrorListInfoBarController_CurrentBackgroundProcessorCancellation()
             this.infoBarManager.AssertHasNoAttachedInfoBar(ErrorListInfoBarController.ErrorListToolWindowGuid);
         }
 
+        [TestMethod]
+        public void ErrorListInfoBarController_Refresh_NonCriticalExceptionIsSuppressed()
```

Comment:
The new tests don't require anything from the existing `TestInitialize` method.
It was simpler just to use mocks and create everything inline than to try to change shared initialization code.